drivers/pcie/host/Kconfig

```diff
@@ -44,6 +44,12 @@ config PCIE_MSI_X
 
 endif # PCIE_MSI
 
+config PCIE_PTM
+	bool "Enable support for PCI(e) Precision Time Management (PTM)"
```

Comment:
Is this `Precision Time Measurement` ?

Reply:
No it's Precision Time Management , at least from PCIe 4.0 r1 specs (section 7.9.16)

Reply:
I think in most of the places they use  name `Measurements` in that document

Reply:
In `PCI Express® Base Specification Revision 5.0 Version 1.0`, the capability name is indeed `Precision Time Management Extended Capability` for controlling `PTM` Hierarchy which means `Precision Time Measurement`.

Reply:
In that document the seems to use both names for that capability, so it should be OK to use `Management`.